docs/source/dataframe-overview.rst

```diff
@@ -76,19 +80,51 @@ What definitely works?
     *  Common aggregations:  ``df.x.max()``, ``df.max()``
     *  Is in:  ``df[df.x.isin([1, 2, 3])]``
     *  Datetime/string accessors:  ``df.timestamp.month``
-* Cleverly parallelizable operations (also fast):
+* Cleverly parallelizable operations (fast):
     *  groupby-aggregate (with common aggregations): ``df.groupby(df.x).y.max()``,
        ``df.groupby('x').max()``
     *  value_counts:  ``df.x.value_counts()``
     *  Drop duplicates:  ``df.x.drop_duplicates()``
     *  Join on index:  ``dd.merge(df1, df2, left_index=True, right_index=True)``
+    *  Join with Pandas DataFrames: ``dd.merge(df1, df2, on='id')``
+    *  Elementwise operations with different partitions / divisions: ``df1.x + df2.y``
+    *  Datetime resampling: ``df.resample(...)``
+    *  Rolling averages:  ``df.rolling(...)``
+    *  Pearson Correlations: ``df[['col1', 'col2']].corr()``
 * Operations requiring a shuffle (slow-ish, unless on index)
     *  Set index:  ``df.set_index(df.x)``
     *  groupby-apply (with anything):  ``df.groupby(df.x).apply(myfunc)``
-    *  Join not on the index:  ``pd.merge(df1, df2, on='name')``
-    *  Elementwise operations with different partitions / divisions: ``df1.x + df2.y``
-* Ingest operations
-    *  CSVs: ``dd.read_csv``
-    *  pandas: ``dd.from_pandas``
-    *  Anything supporting numpy slicing: ``dd.from_array``
-    *  Dask.bag: ``mybag.to_dataframe(columns=[...])``
+    *  Join not on the index:  ``dd.merge(df1, df2, on='name')``
+
+See :doc:`DataFrame API documentation<dataframe-api>` for a more extensive
+list.
+
+
+Execution
+---------
+
+By default ``dask.dataframe`` uses the multi-threaded scheduler.
+This exposes some parallelism when pandas or the underlying numpy operations
+release the global interpreter lock (GIL).  Generally pandas is more GIL
+bound than NumPy, so multi-core speed-ups are not as pronounced for
+``dask.dataframe`` as they are for ``dask.array``.  This is changing, and
+the pandas development team is actively working on releasing the GIL.
+
+In some cases you may experience speedups by switching to the multiprocessing
+or distributed scheduler.
+
+.. code-block:: python
+
+   >>> dask.set_options(get=dask.multiprocessing.get)
```

Comment:
Since this example only shows the multiprocessing scheduler, maybe a link to the scheduler docs page?